### DISCUSSION TITLE
How to collect the metric 'browser_app_page_uv'

### DISCUSSION THREAD

**Discussion by yueguiji** [apache/skywalking#11861] [Q&A] [answered]
In my project , I want to show a table about app uv like browser_app_page_pv.
How could I do.

![image](https://github.com/apache/skywalking/assets/92442632/ae19b1d9-c794-4b58-afae-cca7d080e5e3)

**Comment by wu-sheng** [comment, marked as answer]
We don't do that. UV is very expensive, and not suitable for APM.

**Reply by yueguiji** [reply]
Ok~I understand.

**Comment by tanshanshan** [comment]
You can use customTags to  record UV
